src/krbd.cc

```diff
@@ -192,7 +192,7 @@ static int have_minor_attr(void)
 static int build_map_buf(CephContext *cct, const krbd_spec& spec,
                          const string& options, string *pbuf)
 {
-  bool msgr2 = false;
+  bool msgr2 = true, ms_mode_specified = false;
```

Comment:
Nit: I'd advise against declaring variables with different initialization values together like that.  Unless the variables are _very_ closely related, the fact that they are of the same type doesn't mean that everything should go on the same line ;)
```suggestion
  bool msgr2 = true;
  bool ms_mode_specified = false;
```